### DISCUSSION TITLE
List of AWS Services that are in scope for prowler

### DISCUSSION THREAD

**Discussion by ldina** [prowler-cloud/prowler#1398] [General]
Hi all,
is there a way to understand which AWS services are in scope to receive a prowler check and be included in the inventory?

**Comment by toniblyx** [comment]
To see all services in scope for Prowler see the output of this command below:
`./prowler -l | awk -F"\ -\ " '{ print $2 }' | awk -F"\ " '{ print $1 }' | sort | uniq`

Output of that command for `Prowler 2.11.0-21July2022`:
```
accessanalyzer
acm
apigateway
appstream
autoscaling
cloudformation
cloudfront
cloudtrail
cloudwatch
codeartifact
codebuild
configservice
ds
dynamodb
ec2
ecr
ecs
efs
eks
elb
emr
es
glacier
glue
guardduty
iam
kms
lambda
logs
macie
rds
redshift
route53
s3
sagemaker
secretsmanager
securityhub
shield
sns
sqs
ssm
support
trustedadvisor
vpc
workspaces
```
If you want to understand how the quick inventory works, please read the doc here https://github.com/prowler-cloud/prowler#inventory

**Reply by ldina** [reply]
@toniblyx great, neat trick!
Thanks
Lorenzo
